### PR TITLE
feat(Card): use common constants for card's radiuses

### PR DESCRIPTION
**Pull request by Arucard89**
- add new radius constant --g-border-radius-xxl
- replace string values to common radius constants in styles
source:
https://github.com/gravity-ui/landing/pull/220#discussion_r1657214751

### CODE REVIEW

File: src/components/Card/Card.scss [hunk 1]
```diff
@@ -157,11 +157,11 @@ $block: '.#{variables.$ns}card';
 
     &_size {
         &_m {
-            --_--border-radius: 8px;
+            --_--border-radius: var(--g-border-radius-l);
```
Comment (korvin89) [inline src/components/Card/Card.scss:160]: Why we use `--g-border-radius-l` for `size_m`?

Comment (Arucard89) [reply src/components/Card/Card.scss:160]: Card's border sizes don't have straight accordance to radius constants. I have choosen constants by values.
https://github.com/gravity-ui/uikit/blob/main/styles/themes/common/_index.scss#L13

Comment (korvin89) [reply src/components/Card/Card.scss:160]: @amje I think it is a not good idea to use symantic variables in such cases

Comment (Arucard89) [reply src/components/Card/Card.scss:160]: What do you think about adding variables for cards? 
For example:
—g-border-radius-card-m
—g-border-radius-card-l

Comment (Arucard89) [reply src/components/Card/Card.scss]: > What do you think about adding variables for cards to _index.scss?
> For example:
> —g-border-radius-card-m
> —g-border-radius-card-l

@korvin89 , @amje, I mean like this: https://github.com/gravity-ui/uikit/pull/1682/commits/a50b4604eade6bc6ab47d30fe7d86ceadc100ed6

In my opinion, these variables also helps to make themisation easier, without using CSS API

**Pull request closed without merging**